sonar-project.properties

```
@@ -0,0 +1,36 @@
+# Organization and project keys are displayed in the right sidebar of the project homepage
+sonar.organization=owncloud-1
+sonar.projectKey=owncloud_core
+sonar.projectVersion=1.0
+sonar.host.url=https://sonarcloud.io
+
+# =====================================================
+#   Meta-data for the project
+# =====================================================
+
+sonar.links.homepage=https://github.com/owncloud/core
```

```suggestion
sonar.links.homepage=https://github.com/owncloud/data_exporter
```
change all these core to data_exporter

---

done